### DISCUSSION TITLE
ant task multi-threading support

### DISCUSSION THREAD

**Discussion by tnaroska** [pmd/pmd#3806] [Q&A] [answered]
Hi,
there seems to be a hidden 'threads' parameter on the ant pmd task here:
https://github.com/pmd/pmd/blob/master/pmd-core/src/main/java/net/sourceforge/pmd/ant/PMDTask.java#L33

It does not show up on the task documentation [page](https://pmd.github.io/latest/pmd_userdocs_tools_ant.html#parameters).

Is this a supported feature?

**Comment by adangel** [comment, marked as answer]
I think, there is just documentation missing. The value is passed on when executing PMD, so it is supported.
Note: if not set, it currently defaults to 0 or 1, so currently PMD is run single threaded by ant.

**Reply by adangel** [reply]
I'll fix the doc for PMD 6.43.0 (#3807)
